src/jaxsim/mujoco/visualizer.py

```diff
@@ -97,11 +200,11 @@ def record_frame(self, camera_name: str = "track") -> None:
         frame = self.render_frame(camera_name=camera_name)
         self.frames.append(frame)
 
-    def write_video(self, path: pathlib.Path, exist_ok: bool = False) -> None:
+    def write_video(self, path: pathlib.Path | str, exist_ok: bool = False) -> None:
```

Comment:
This is also out of the scope of the PR. Change the description  of the PR accordingly